src/Features/CSharp/Portable/Ambiguity/CSharpAmbiguousTypeCodeFixProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.AmbiguityCodeFixProvider;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.Ambiguity
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.AliasAmbiguousType), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.FullyQualify)]
+    internal class CSharpAmbiguousTypeCodeFixProvider : AbstractAmbiguousTypeCodeFixProvider
```

Comment:
type name should match provider name.  if provider name is AliasAmbiguousType, type names shoudl be CSharpAliasAmbiguousTypeCodeFixProvider and so on